core/whistle-1.0.0/src/wh_nodes.erl

```diff
@@ -415,10 +415,8 @@ handle_cast({'advertise', JObj}, #state{tab=Tab}=State) ->
 handle_cast({'gen_listener', {'is_consuming', 'true'}}
             ,#state{heartbeat_ref='undefined'}=State
            ) ->
-    Reference = erlang:make_ref(),
-    erlang:send_after(?HEARTBEAT, self(), {'heartbeat', Reference}),
     lager:debug("start heartbeat"),
-    {'noreply', State#state{heartbeat_ref=Reference, is_consuming='true'}};
+    handle_info({'heartbeat', 'undefined'}, State#state{is_consuming='true'});
```

Comment:
@siplabs I am ready to accept this PR (thanks what a tremendous amount of work, I look forward to playing with it!)  However, this change to wh_nodes bypasses the saftey check to make sure there are not more than one timer running at a time.  If there is a bug or something broken here lets talk but I don't think it is related to this PR.  If you remove the change to wh_nodes we could accept this!